src/lib/mongoBackend/TriggeredSubscription.cpp

```diff
@@ -121,3 +128,14 @@ std::string TriggeredSubscription::toString(const std::string& delimiter)
 
   return ss.str();
 }
+
+
+
+/* ****************************************************************************
+*
+* TriggeredSubscription::stringFilterSet - 
+*/
+void TriggeredSubscription::stringFilterSet(StringFilter* _stringFilterP)
```

Comment:
I understand this function is used when the string filter is populated into triggered subscription from cache. Is that correct?